web/app.js

```diff
@@ -2179,7 +2179,7 @@ window.addEventListener('afterprint', function afterPrint(evt) {
 
 exports.PDFViewerApplication = PDFViewerApplication;
 
-// TODO remove circular reference of pdfjs-web/secondary_toolbar on app.
+//// TODO remove circular reference of pdfjs-web/secondary_toolbar on app.
```

Comment:
Needed because our preprocessor removes this comment -- was not an issue since in the past it was a two stage preprocessing.